plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -425,6 +425,14 @@
         option has a default of C(no). Note that the default value of I(proxmox_default_behavior)
         changes in community.general 4.0.0.
     type: bool
+  tags:
+    description:
+      - List of tags to apply to the VM instance.
+      - Tags must start with [a-z0-9_] followed by zero or more of the following characters [a-z0-9_-+.].
```

Comment:
The following uses `<code>...</code>` to format the regexes:
```suggestion
      - Tags must start with C([a-z0-9_]) followed by zero or more of the following characters C([a-z0-9_-+.]).
```